css/fonts.css

```diff
@@ -0,0 +1,7 @@
+@font-face {
+  font-family: 'PT Serif';
+  src: url('.fonts/ptserif-bold-webfont.woff2') format('woff2'), 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/html_abrmhah/pullRequest?prid=6295173)